api/src/main/resources/Covid-19_Concepts-1.xml

```diff
@@ -0,0 +1,521 @@
+<concept_reference_source concept_source_id="2" name="SNOMED NP" description="Non-preferred SNOMED CT mappings" creator="1" date_created="2010-04-25 11:12:44.0" retired="false" retired_by="1" uuid="2ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
+<concept_reference_source concept_source_id="3" name="ICD-10-WHO" description="ICD-10 WHO Version" creator="1" date_created="2010-04-25 11:13:15.0" retired="false" retired_by="1" uuid="3f65bd34-26fe-102b-80cb-0017a47871b2" changed_by="1"/>
+<concept_reference_source concept_source_id="6" name="LOINC" description="LOINC code" creator="1" date_created="2010-04-25 11:14:13.0" retired="false" retired_by="1" uuid="6ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
+<concept_reference_source concept_source_id="8" name="ICD-10-WHO 2nd" description="Preferred secondary map to ICD-10-WHO" creator="1" date_created="2010-04-25 12:18:19.0" retired="false" retired_by="1" uuid="8ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
+<concept_reference_source concept_source_id="9" name="ICD-10-WHO NP2" description="Non-preferred secondary map to ICD-10-WHO" creator="1" date_created="2010-04-25 12:18:41.0" retired="false" retired_by="1" uuid="9ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
+<concept_reference_source concept_source_id="19" name="3BT" description="Belgian Bilingual Bi-encoded Thesaurus (3BT)" creator="1" date_created="2013-02-16 22:25:25.0" retired="false" retired_by="1" uuid="19ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
+<concept_reference_source concept_source_id="20" name="ICPC2" description="International Classification of Primary Care version 2 from WONCA." creator="1" date_created="2013-02-17 15:25:41.0" retired="false" retired_by="1" uuid="20ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
+<concept_reference_source concept_source_id="21" name="CIEL" description="Columbia International eHealth Laboratory concept ID" creator="1" date_created="2013-07-03 15:55:03.0" retired="false" retired_by="1" uuid="21ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
+<concept_reference_source concept_source_id="24" name="IMO ProblemIT" description="Intelligent Medical Objects, Inc. problem/disease/finding datasource" creator="1" date_created="2013-10-01 13:34:15.0" retired="false" retired_by="1" uuid="24ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
+<concept_reference_source concept_source_id="25" name="IMO ProcedureIT" description="Intelligent Medical Objects, Inc. procedure/test datasource" creator="1" date_created="2013-10-01 13:34:40.0" retired="false" retired_by="1" uuid="25ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
+<concept_reference_source concept_source_id="38" name="SNOMED UK" description="SNOMED CT UK extension" creator="1" date_created="2020-03-05 19:08:04.0" retired="false" retired_by="1" uuid="38ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
```

Comment:
A good number of these reference sources are already in the default RefApp metadata, although I'm not sure how they are loaded.

Reply:
true @ibacher , some of them are already loaded from the Reference_Application_Concepts-25.xml file

Reply:
let me clear them

Reply:
@ibacher , i removed the existing reference sources